server/middlewares/stats.js

```diff
@@ -48,6 +51,36 @@ const incrCountersBy = (key, value) => {
   client.incrby(`${key}:daily:${year}-${month}-${day}`, value)
 }
 
+const initVisitor = (trackingID, userID) => {
+  return ua({
+    tid: trackingID,
+    cid: userID,
+    strictCidFormat: false,
```

Comment:
if this is really needed, it would be nice to put there a comment and explain the reason behind why this should be false, for future contributors